atom/api.py

```diff
@@ -1,10 +1,13 @@
 #------------------------------------------------------------------------------
-# Copyright (c) 2013, Nucleic Development Team.
+# Copyright (c) 2013-2017, Nucleic Development Team.
 #
 # Distributed under the terms of the Modified BSD License.
 #
 # The full license is in the file COPYING.txt, distributed with this software.
 #------------------------------------------------------------------------------
+from __future__ import (division, unicode_literals, print_function,
+                        absolute_import)
```

Comment:
I noticed that you have this in every module in the `atom` package. However I don't think it's needed:
* nothing is using the divide operator
* nothing is printing (examples are another story...)
* `absolute_import`: optional after Python 2.5.0a1, and this project already uses absolute imports
* `unicode_literals`: optional after Python 2.6.0a2, only used in `scalars.py`

See: https://docs.python.org/2/library/__future__.html

I'm not sure what versions of Python should be supported by atom, but it might be OK to not worry about 2.5 & 2.6 support.